package.json

```diff
@@ -14,10 +14,10 @@
     "rollup-plugin-terser": "^7.0.2"
   },
   "dependencies": {
-    "@duckduckgo/autoconsent": "^3.0.4",
+    "@duckduckgo/autoconsent": "^4.1.0",
     "@duckduckgo/autofill": "github:duckduckgo/duckduckgo-autofill#6.1.1",
     "@duckduckgo/content-scope-scripts": "github:duckduckgo/content-scope-scripts#3.4.1",
-    "@duckduckgo/privacy-dashboard": "github:duckduckgo/privacy-dashboard#1.3.0",
+    "@duckduckgo/privacy-dashboard": "github:muodov/privacy-dashboard#f9b07e91a91c7976a8fcb990353da4961c4d480d",
```

Comment:
what's the plan here? I assume we want to use a release version before merging?

Reply:
+1, we can't merge this PR pointing to anything that is not `duckduckgo` repo

Reply:
absolutely, I am going to replace https://github.com/duckduckgo/Android/pull/2738/commits/e884025017e089fbf1d2029866093e1acb9d36eb once the corresponding [dashboard PR](https://github.com/duckduckgo/privacy-dashboard/pull/21) is merged

Reply:
PD is released, I've pinned it to the production version